browser/src/layer/tile/ScrollSection.ts

```diff
@@ -608,7 +609,8 @@ class ScrollSection {
 			this.sectionProperties.drawHorizontalScrollBar = true;
 			this.sectionProperties.mouseIsOnHorizontalScrollBar = true;
 			this.sectionProperties.mapPane.style.cursor = 'pointer';
-			this.containerObject.requestReDraw();
+			if (!this.containerObject.draggingSomething && !(<any>window).mode.isDesktop())
+				this.containerObject.requestReDraw();
```

Comment:
on mobile when dragging "scroll indicator handler (circle with arrows up/down) and view goes outside currently rendered view - we get blank screen. It is rendered only when we finish dragging. Is that expected?

Reply:
no, it works when tried again :)